components/clp-py-utils/clp_py_utils/s3_utils.py

```diff
@@ -51,16 +53,16 @@ def _get_session_credentials(aws_profile: Optional[str] = None) -> Optional[S3Cr
     )
 
 
-def get_credential_env_vars(config: S3Config) -> Dict[str, str]:
+def get_credential_env_vars(auth: AwsAuthentication) -> Dict[str, str]:
     """
     Generates AWS credential environment variables for tasks.
-    :param config: S3Config or S3InputConfig from which to retrieve credentials.
-    :return: A [str, str] Dict which access key pair and session token if applicable.
-    :raise: ValueError if auth type is not supported
+    :param auth: AwsAuthentication
+    :return: A [str, str] Dict containing access key pair and session token, or an empty dictionary
+    if the AWS credential environment variables are supposed to be already set.
```

Comment:
```suggestion
    :return: A dictionary containing an access key-pair and optionally, a session token; or an empty
    dictionary if the AWS-credential environment-variables should've been set already.
```

The return type already indicates it's a Dict[str, str], so removed that from the docstring. Also, tried to clarify it a bit. Is it still accurate?

Reply:
yeah, it's still accurate, thanks!